truenas_install/__main__.py

```diff
@@ -363,6 +363,8 @@ def main():
 
                     setup_machine_id = configure_serial = True
 
+                run_command(["chmod", "700", f"{root}/data"])
```

Comment:
Seems you missed -R

Reply:
Or perhaps `chmod -R u=rwX,g=,o= <path>`
Which will set  0700 on dirs and 0600 on files.

Reply:
@anodos325 just to confirm will that syntax work on FreeBSD?

Reply:
```
root@homenas[/mnt/dozer]# chmod -R u=rwX,g=,o= f
root@homenas[/mnt/dozer]# stat -x f/testfile    
  File: "f/testfile"
  Size: 0            FileType: Regular File
  Mode: (0600/-rw-------)         Uid: (    0/    root)  Gid: (    0/   wheel)
Device: 18446744072137047106,1457848327   Inode: 426380    Links: 1
Access: Mon Aug 29 13:36:04 2022
Modify: Mon Aug 29 13:36:04 2022
Change: Mon Aug 29 13:36:54 2022
root@homenas[/mnt/dozer]# cat /etc/version 
TrueNAS-12.0-U8.1 (4e9efc861d)
```

Reply:
Do we want to recursively set permissions as 700 for all files?
This will set executable bit for database as well..

Reply:
> Do we want to recursively set permissions as 700 for all files? This will set executable bit for database as well..

See my chmod incantation above. It sets 700 for dirs and 600 for files. `X` vs `x`.